cli/build.gradle.kts

```diff
@@ -17,8 +17,6 @@
  * License-Filename: LICENSE
  */
 
-import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
```

Comment:
> Fixes #6318.

Tiny commit message nit: This does not really *fix* #6318, but *resolves* (or just *closes*) the issue in another way than fixing it.